### ISSUE TITLE
Download a dat from the browser

### ISSUE THREAD

**Issue by karissa** [datproject/datproject.org#560]
It'd be nice to be able to download a dat directly from the browser without making the other user download anything. Found this out through user testing.

However, what are our bandwidth costs associated with this? How can we do this cheaply or for free and when does it break things?

**Comment by clkao** [comment]
note that back in the webrtc transport we did have this, by having the client create a zip on the fly from the archive content retrieved from hyperdrive.

**Comment by karissa** [comment]
Yeah, it did it in memory on the client side so the performance was capped at a certain level.
